### DISCUSSION TITLE
Image in table cell

### DISCUSSION THREAD

**Discussion by webwurm** [Budibase/budibase#6429] [Help]
I pull my data from Firestore and have URLs to pictures in there - in my case, profile-pictures.

Now I want to show the images in the table. How can I do that it Budibase? I can't find anything in the docs or in the discussions.

![Photo_Url](https://user-images.githubusercontent.com/1053176/175034410-cc94653b-f0f0-4dbe-9050-0e3ca22e1c04.jpg)

Sorry if it is a trivial question, it is my 2nd day learning Budibase :)

**Comment by melohagan** [comment]
Hey @webwurm 

This is actually a little awkward. At the moment, images within table cells are part of attachments (https://docs.budibase.com/docs/attachments)

As you are just storing the image URLs yourself, they won't appear as images in the table cells. I'm not sure if there is a "hack" around this.

If however, you are content to have the image at the start of the row, then you can try this:

1. Select your table
2. Select Elements > Image
    ![Screenshot 2022-06-22 at 17 02 17](https://user-images.githubusercontent.com/101575380/175078596-aa75f1a6-a88e-4960-a8f8-92a177493db0.png)
3. Hit the dropdown on the URL. Look for your <table-name>.photo_url_thumbnail
![Screenshot 2022-06-22 at 17 03 13](https://user-images.githubusercontent.com/101575380/175078884-811d453e-752e-46e4-aab9-b3ee717bef69.png)

**Reply by webwurm** [reply]
Thanks a lot, but I can't get it to work. 
When I click on the table > Element > Image then the image appears in the element-tree under the table - and I can't access the table-columns from there:
<img width="428" alt="Bild 1" src="https://user-images.githubusercontent.com/1053176/175095093-753066ba-ad90-4484-9cae-5bc2b20e7db6.png">

Am I doing something wrong?

**Reply by melohagan** [reply]
Ah sorry, that's probably because you're using a Table Block. If you use a Data Provider with a Table instead, then you can add images to the Table component. 

- https://docs.budibase.com/docs/table
